packages/elements-next/src/index.mdx

```diff
@@ -7,10 +7,105 @@ route: /
+- Utilities mixed in with React components - both a poor separation of concens and insufficient documentation for the utilities.
+
+### Specification
+
+- Elements is a UI only library. Utilities and other modules should be migrated to the `@reapit/utils` package and will be documented and released as a separate NPM package.
+- CSS should be handwritten, we will not import any 3rd party CSS so we have full control over styles unless absolutely necessary eg DatePicker.
+- We should keep NPM packages to an absolute bare minimum, no packages to achieve trivial tasks to reduce library bloat.
+- All components to be strongly typed and heavily tested
+- Code that can be migrated or repurposed from v1 should be however, every component should be viewed critically, improved, upgraded, tidied and made consistent.
+- Components should be flexible where possible, render children to allow styling for custom components is a desirable pattern rather than binding specific single-use behaviours to the UI
+- Props should be consistent, boolean and imperative modifiers to allow for clean markup. These booleans should map to classes, re-usable where possible.
+
+```
+isActive
+isFlex
+isFlexColumn
+isTextCentered
+isPrimary
+isDanger
+isFullWidth
+hasChildren
+```
+- The generic styles folder contains a lib of generic classes, these should be used and composed over writing new CSS
+- All classes should be prefixed with `el` to ensure we do not clash with third party libs eg `elIsFlex` in Linaria will generate the class `el-is-flex`.
+- Colors, margins, borders, fonts, padding, font-sizes and anything that can be vairablized should be. We want to be able to change a global variable and see the changes everywhere.
+- CSS should look the same as the Bulma equivient where the component is equivient. Eg an input or a button in v1 should look identical in v2. We are actively hiring a designer and the styles will change in time but we want this to be simple to do when the time comes.
+- The docs should be styled the same as Elements - Elements docs should look like Elements!
+- There should be far fewer components. We have several different ways of doing notifications - there should be a single approach. See below for list of components to migrate.
+- HTML output should be next to the mounted React Component and the Rendered Component in the docs in all cases.
+- All variants of components should be visible at the top of the page in the generated props section. They should all be editable at the bottom, although only the basic component without variants should be rendered by default.
+- Where we export multiple components from the same file, we should export them as modules eg `<Headings.Subheading>`
+- Should work in Edge (latest version before Chromium), Edge Chromium, Firefox, Chrome, Safari
```

Comment:
Just a note, I think we should add `postcss` in the build process to include `https://github.com/postcss/autoprefixer`, so that all the styles will not break in different browser.
There's a few postcss pluggin you may want to check out, like [cssnano](https://cssnano.co/)
Look like linaria supported `PostCSS` it too 

![Screen Shot 2020-09-07 at 11 07 00](https://user-images.githubusercontent.com/26746668/92347452-582c2600-f0fa-11ea-98d6-9aed606da6c1.png)

Reply:
Yep, agreed, we should definitely do this.